app/_src/policies/meshmetric.md

```diff
@@ -0,0 +1,284 @@
+---
+title: MeshMetric
+---
+
+{% warning %}
+This policy uses new policy matching algorithm.
+Do **not** combine with [Traffic Metrics](/docs/{{ page.version }}/policies/traffic-metrics).
+{% endwarning %}
+
+{{site.mesh_product_name}} facilitates consistent traffic metrics across all data plane proxies in your mesh.
+
+You can define metrics configuration for a whole Mesh, and optionally tweak certain parts for individual data plane proxies.
+For example, you might need to override the default metrics port if it's already in use on the specified machine.
+
+{{site.mesh_product_name}} provides full integration with Prometheus:
+
+* Each proxy can expose its metrics in [Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format).
+* {{site.mesh_product_name}} exposes an API called the monitoring assignment service (MADS) which exposes proxies configured by `MeshMetric`.
+
+To collect metrics from {{site.mesh_product_name}}, you need to expose metrics from proxies and applications.
+
+
+{% tip %}
+In the rest of this page we assume you have already configured your observability tools to work with {{site.mesh_product_name}}.
+If you haven't already read the [observability docs](/docs/{{ page.version }}/explore/observability).
+{% endtip %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to | from |
+|-------------------|-----------|----|------|
+| Mesh              | ✅         | ❌  | ❌    |    
+| MeshSubset        | ✅         | ❌  | ❌    |    
+| MeshService       | ✅         | ❌  | ❌    |    
+| MeshServiceSubset | ✅         | ❌  | ❌    |    
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+There are three main sections of the configuration: `sidecar`, `applications`, `backends`.
+The first two define how to scrape parts of the mesh (sidecar and underlying applications), the third one defines what to do with the data (in case of Prometheus instructs to scrape specific address, in case of OpenTelemetry defines where to push data).
+
+{% tip %}
+In contrast to [Traffic Metrics](/docs/{{ page.version }}/policies/traffic-metrics) all configuration is dynamic and no restarts of the DPPs are needed.
```

Comment:
I think we should use the full names whenever the abbreviation is not generally well known
```suggestion
In contrast to [Traffic Metrics](/docs/{{ page.version }}/policies/traffic-metrics) all configuration is dynamic and no restarts of the Data Plane Proxies are needed.
```